app/controllers/v0/facilities/health_controller.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-class V0::Facilities::VaController < FacilitiesController
+class V0::Facilities::HealthController < FacilitiesController
   before_action :validate_params, only: [:index]
```

Comment:
it looks like both CemeteryController and HealthController require validate_params on index, would it be worth putting in your base FacilitiesController?